Changelog.md

```diff
@@ -0,0 +1,59 @@
+# Toolkit Changelog
+
+## Process for updating this change log (Proposal)
+This change log should be updated as part of a PR if the work is worth noting (most of them should be). If unsure, always add an entery here for any PR targeted for the next release. It's easier to remove than add an entry at final review time for the next release.
+
+## Next Release (Date TBD)
+
+#### New experimental features
+
+#### Features stabilized
+
+#### Bug fixes
+- [#568](https://github.com/timescale/timescaledb-toolkit/pull/568): Allow `approx_count` accessor function to take NULL inputs.
```

Comment:
#574 was merged since you created this PR, it should be added to the changelog:

```suggestion
#### Bug fixes
- [#568](https://github.com/timescale/timescaledb-toolkit/pull/568): Allow `approx_count` accessor function to take NULL inputs.
- [#574](https://github.com/timescale/timescaledb-toolkit/pull/574): Add default unit to interpolated_integral.
```

Reply:
Thanks.